### ISSUE TITLE
Warning bar not announced by screen reader

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6963]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.959
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- start screen reader and load RStudio Server
- in console: rstudioapi::executeCommand("showWarningBar")
- this shows warning bar with text "This is a warning!"

### Describe the problem in detail
- the warning bar text is not announced

### Describe the behavior you expected
- warning bar text is announced otherwise real warnings using this widget are invisible to screen reader users

### Cause
If you turn off warning bar announcement in Tools / Global Options / Accessibility / Announcements then it starts working, so the logic is inverted.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by R-Hannibal** [comment]
@jooyoungseo - would it be possible for you to verify this issue is fixed in the latest 1.3 patch preview build (available [HERE](https://rstudio.com/products/rstudio/download/preview/))?

**Comment by gtritchie** [comment]
@jooyoungseo This version (1.3.1023) is installed on the original accessibility test server.

**Comment by jooyoungseo** [comment]
@R-Hannibal, Perfectly resolved in v1.3.1023.
